crates/ruff_cache/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "ruff_cache"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
I've been using this pattern in the other internal-only crates:

```toml
[package]
name = "ruff_cache"
version = "0.0.0"
publish = false
edition = { workspace = true }
rust-version = { workspace = true }
```

Thoughts?

Reply:
I should not have used the CLion standard template...